src/app/components/Sidebar/Sidebar.jsx

```diff
@@ -10,24 +10,33 @@ import config from '../../../../appConfig';
 const fieldsetProps = {
   season: {
     fieldsetName: 'season',
+    currentValue: '',
     options: [
-      { name: '2018 Spring', value: '2018-03-01' },
-      { name: '2018 Winter', value: '2018-01-01' },
-      { name: '2017 Fall', value: '2017-09-01' },
-      { name: '2017 Summer', value: '2017-06-01' },
+      { name: '2018 Spring', value: '2018-03-01', disabled: false },
+      { name: '2018 Winter', value: '2018-01-01', disabled: false },
+      { name: '2017 Fall', value: '2017-09-01', disabled: false },
+      { name: '2017 Summer', value: '2017-06-01', disabled: false },
     ],
   },
   audience: {
     fieldsetName: 'audience',
+    currentValue: '',
     options: [
-      { name: 'Adult', value: 'adult' },
-      { name: 'Teen', value: 'ya' },
-      { name: 'Children', value: 'children' },
+      { name: 'Adult', value: 'Adult', disabled: false },
+      { name: 'Teen', value: 'YA', disabled: false },
+      { name: 'Children', value: 'Children', disabled: false },
     ],
   },
 };
 
 const Sidebar = (props) => {
+  const updateCurrentListSelectorValue = (data) => {
```

Comment:
The issue I have with this is that you are updating an object which exists outside of the component. I think you'll cover/update this in your new PR when you get a better list of data, but be aware that you should try not to mutate the original object. Try creating a new object in your next PR instead.